tools/alpine/versions

```diff
@@ -0,0 +1,162 @@
+alpine-baselayout-3.0.4-r0
```

Comment:
yes, this is what we used to have for the alpine base image, it was useful to see what the security updates were, I will add a note to the top.